build.py

```diff
@@ -1,18 +1,69 @@
-import math
-import subprocess
 from pathlib import Path
 
+import argparse
+
 import fontmake.instantiator
 import fontTools.designspaceLib
 import ufo2ft
 import ufoLib2
 import vttLib
-import copy
 
 INPUT_DIR = Path("sources")
 OUTPUT_DIR = Path("build")
+VTT_DATA_FILE = (INPUT_DIR / "vtt_data" / "CascadiaCode.ttx")
+
+def step_set_font_name(n):
+    def _set(instance):
+        instance.info.familyName = n
+    return _set
+
+def step_merge_glyphs_from_ufo(path):
+    def _merge(instance):
+        ufo = ufoLib2.objects.font.Font.open(path)
+        print (f"[{instance.info.familyName}] Merging {path}")
+        for glyph in ufo.glyphOrder:
+            if glyph not in instance.glyphOrder:
+                instance.addGlyph(ufo.get(glyph))
+
+    return _merge
+
+def step_set_feature_file(n):
+    with open(n, 'r') as feaCode:
+        fea = feaCode.read()
+    def _set(instance):
+        instance.features.text = fea
```

Comment:
```suggestion
        instance.features.text = n.read_text()
```

Reply:
Chose to keep `fea` captured into the lambda to avoid re-reading the file from disk 2 additional times. Consider it memoized! :grin: